dash/testing/browser.py

```diff
@@ -81,16 +90,20 @@ def take_snapshot(self, name):
             "{}/{}_{}.png".format(target, name, self.session_id)
         )
 
-    def find_element(self, css_selector):
-        """wrapper for find_element_by_css_selector from driver"""
-        return self.driver.find_element_by_css_selector(css_selector)
+    def find_element(self, selector):
+        """find_element returns the first found element by the css `selector`
+        shortcut to `driver.find_element_by_css_selector`
+        """
+        return self.driver.find_element_by_css_selector(selector)
 
-    def find_elements(self, css_selector):
-        """wrapper for find_elements_by_css_selector from driver"""
-        return self.driver.find_elements_by_css_selector(css_selector)
+    def find_elements(self, selector):
+        """find_elements return a list of matching elements by the css
+        `selector` shortcut to `driver.find_elements_by_css_selector`
```

Comment:
find_elements returns a list of all elements matching the css
`selector`. shortcut to `driver.find_elements_by_css_selector`